examples/resnet/resnet_main.py

```diff
@@ -133,21 +126,15 @@ def __init__(self, data, dataset, eval_batch_count, eval_dir):
             relu_leakiness=0.1,
             optimizer="mom",
             num_gpus=0)
-        input_images = data[0]
-        input_labels = data[1]
         with tf.device("/cpu:0"):
             # Builds the testing network.
-            images, labels = cifar_input.build_input([input_images,
-                                                      input_labels],
+            images, labels = cifar_input.build_input(data,
                                                      hps.batch_size, dataset,
                                                      False)
             self.model = resnet_model.ResNet(hps, images, labels, "eval")
             self.model.build_graph()
-            config = tf.ConfigProto(allow_soft_placement=True)
```

Comment:
I guess `allow_soft_placement` never did anything in this case because it's just using CPUs, is that right?

Reply:
Correct. `allow_soft_placement` was done because certain operations in tensorflow simply cannot be placed on the GPU.